lib/src/predicate.dart

```diff
@@ -27,7 +29,7 @@ class FieldPredicate<T> {
 
     result.add(_operand);
 
-    if (_value != null) {
+    if (_useValue) {
```

Comment:
I don't understand the reasoning behind the introduction of this property: `_useValue`

Reply:
The reasoning is that I need to be able to use `null` as an output value, so I need this `_useValue` variable to be able to distinguish between `_value` is `null` and I want to write `null` in the output, or `_value` is null, but I don't want to output it

Reply:
Understood.  Then it's all good to go.